src/api/plugins/tdbp/warren_tdbp/api.py

```diff
@@ -23,13 +26,14 @@
 
 @router.get("/window")
 async def get_sliding_window(
-    course_id: str = Query(
-        description="The course identifier on Moodle",
-    ),
-    until: Optional[date] = Query(
-        description="End date until when to compute the sliding window",
-    ),
-):
+    course_id: Annotated[str, Depends(get_lti_course_id)],
```

Comment:
Can you keep the description ? It is used for API documentation

Reply:
`course_id` and `user_id` are no longer query parameters, but are internally taken from the JWT token forged from the LTI token.
I agree that the token should appear in the documentation though.
When reading the [documentation](https://fastapi.tiangolo.com/tutorial/dependencies/#integrated-with-openapi), and the [utils function](https://github.com/openfun/warren/blob/d64132c60a445d57cd86739df083061338577453/src/api/core/warren/utils.py#L21) on Warren side, the token should be appearing on the OpenAPI doc, but it does not and I'm not sure why...

Reply:
Opened issue on Warren https://github.com/openfun/warren/issues/261